test/Microsoft.Dnx.CommonTestUtils/TestUtils.cs

```diff
@@ -272,6 +272,42 @@ public static DisposableDir PrepareTemporarySamplesFolder(string runtimeHomeDir)
             return tempDir;
         }
 
+        public static DisposableDir CreateDisposableTestProjectFromMisc(string runtimeHomeDir, params string[] projectPaths)
+        {
+            var resultDir = new DisposableDir();
+
+            // Find the misc project to copy
+            var paths = new List<string>(projectPaths);
+            paths.Insert(0, GetMiscProjectsFolder());
+            CopyFolder(Path.Combine(paths.ToArray()), Path.Combine(resultDir, projectPaths[projectPaths.Length - 1]));
+
+            // Make sure package restore can be successful
+            var currentDnxSolutionRootDir = ProjectResolver.ResolveRootDirectory(Directory.GetCurrentDirectory());
+            var nugetConfigName = "NuGet.Config";
+            File.Copy(Path.Combine(currentDnxSolutionRootDir, nugetConfigName), Path.Combine(resultDir, nugetConfigName));
+
+            // Use the newly built runtime to generate lock files for samples
+            string stdOut, stdErr;
+            int exitCode;
+            foreach (var projectDir in Directory.EnumerateDirectories(resultDir))
+            {
+                exitCode = DnuTestUtils.ExecDnu(
+                    runtimeHomeDir,
+                    subcommand: "restore",
+                    arguments: projectDir,
+                    stdOut: out stdOut,
+                    stdErr: out stdErr);
+
+                if (exitCode != 0)
+                {
+                    Console.WriteLine(stdOut);
```

Comment:
`DnuTestUtils.ExecDnu()` is already printing out stdout and stderr to console for you:
https://github.com/aspnet/dnx/blob/dev/test/Microsoft.Dnx.CommonTestUtils/TestUtils.cs#L91

I guess it's pointless to dup the output to console here.